agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -1581,6 +1585,11 @@ bool main_thread::handle_1905_discovery_query(ieee1905_1::CmduMessageRx &cmdu_rx
     //       agent certification
     return true;
 }
+bool main_thread::handle_1905_ap_capability_query(ieee1905_1::CmduMessageRx &cmdu_rx)
+{
+    LOG(DEBUG) << "Received AP Capability Query message";
```

Comment:
Add TODO comment here that explains what needs to be done - send the query to all radio agents. Each agent will then respond with its capabilities, and eventually will be sent to the controller.

Reply:
This is wrong, as said before, the handler should not be here.
CMDUs are passed automatically to the son_slaves.